packages/pglite/src/live/index.ts

```diff
@@ -466,9 +608,27 @@ async function getTablesForView(
         )
         AND d.deptype = 'n';
       `,
-      [viewName],
+      [currentViewName],
     )
-  ).rows.filter((row) => row.table_name !== viewName)
+
+    for (const row of result.rows) {
+      if (row.table_name !== currentViewName && !row.is_view) {
+        const tableKey = `"${row.schema_name}"."${row.table_name}"`
+        if (!tables.has(tableKey)) {
+          tables.set(tableKey, {
+            table_name: row.table_name,
+            schema_name: row.schema_name,
+          })
+        }
+      } else if (row.is_view) {
+        await getTablesRecursive(row.table_name)
```

Comment:
Do we have tests already covering this code path? Given that it involves recursion it'd be good to make sure we don't end up in any weird situations